en/more-08.md

```diff
@@ -277,6 +466,56 @@ would allow the use of `B` in tabular preambles to specify a bold
 centered column.
 
 
+## Vertical tricks
+
+Oftentimes one might think it's a good idea to merge cells vertically to get
+some desired output, but instead one should have split a cell into several
+vertically. With this small trick one can get pretty neat things done. Splitting
+cells into many vertically can be done by nesting a `tabular` environment in a
+single cell:
```

Comment:
This doesn't read naturally at all.  Perhaps

Often, rather than making a cell span multiple rows it is better to instead have a single row in which some cells are split vertically by the use of nested `tabular` environments.